src/components/Login/Login.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import Input from "../Input/Input";
+import styles from "./Login.module.css";
+import Button from "../Button/Button";
+import { Link } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { login } from "../../redux/actions";
+import { useNavigate } from "react-router-dom";
+
+export default function Login() {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
+  const [name, setName] = useState("");
+
+  const signIn = () => {
+    const action = login(name);
+    dispatch(action);
+    navigate("/todo");
+  };
+
+  return (
+    <div className={styles.form}>
+      <Input placeholder={"Enter your name"} value={name} onChange={setName} />
```

Comment:
You can use constants for input's placeholders and button's names.

Reply:
fixed